.github/workflows/publish-packages.yml

```
@@ -0,0 +1,98 @@
+  validate:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.extract_version.outputs.VERSION }}
+    strategy:
+      matrix:
+        package:
+          - name: "prompt-shared-state"
+            path: "prompt-shared-state"
+          - name: "prompt-ui-components"
+            path: "prompt-ui-components"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)